app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>New Recipe</h1>
+
+<%= render 'form', recipe: @recipe %>
```

Comment:
- [ ] As pointed out by the previous reviewer, let's kindly ensure that if we have two users: **A** and **B**, If user **A** create a recipe, user **B** should see all recipes created by **A** but can't delete them. Right now when user **A** create a recipe, user **B** can't even see it.

Reply:
there is a link called public recipes for that, that wasn't mentioned by previous reviewer...